chicago/permit_cleaning.py

```diff
@@ -610,11 +611,8 @@ def deduplicate_permits(cursor, df, start_date, end_date):
     return true_new_permits
 
 
-def gen_file_base_name():
-    today = datetime.today().date()
-    today_string = today.strftime("%Y_%m_%d")
-    file_name = today_string + "_permits_"
-    return file_name
+def gen_file_base_name(start_date, end_date):
+    return f"{start_date}_to_{end_date}_permits_"
```

Comment:
Motivation: It can be hard to remember what the date range was for an export, and usually I have to check the message history of the request to be sure. This change preserves the date range in the filename.